### PR TITLE
fix: markdown headings

### PR DESCRIPTION
**Pull request by eric-muchiri**
Fixed heading issues: spaces around headings

### CODE REVIEW

**Comment by faruktoptas** [comment]
Thanks @eric-muchiri 
Can you reference which issue is fixed?

**Comment by faruktoptas** [comment]
Issues on README.md are listed here:
https://app.codacy.com/manual/faruktoptas/FancyShowCaseView/file/39286766552/issues/source?bid=14909056&fileBranchId=14909056

**Comment by eric-muchiri** [comment]
issue addressed: [no-consecutive-blank-lines] Remove 1 line before node line 20

File: README.md [hunk 1]
```diff
@@ -16,8 +16,8 @@
         <a href="https://material.uplabs.com/posts/fancyshowcaseview">@MaterialUp Best of the Day</a>
 </p>
 
-
```
Comment (faruktoptas) [inline README.md]: this is ok. But other additional lines looks unnecessary.

Comment (eric-muchiri) [reply README.md]: ooh, sorry I thought they were errors. I use markdown lint vs code extension and it highlighted them as errors

Comment (faruktoptas) [reply README.md]: You can check errors from codacy:
https://app.codacy.com/manual/faruktoptas/FancyShowCaseView/file/39286766552/issues/source?bid=14909056&fileBranchId=14909056

Comment (eric-muchiri) [reply README.md]: Sure. I guess a new commit removing added lines will do?

Comment (faruktoptas) [reply README.md]: Yes please

**Pull request merged**